Mage.Sets/src/mage/cards/m/MutatedCultist.java

```diff
@@ -0,0 +1,166 @@
+package mage.cards.m;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.SpellAbility;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.common.RemoveAllCountersSourceCost;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.StaticValue;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CastSourceTriggeredAbility;
+import mage.abilities.effects.common.cost.CostModificationEffectImpl;
+import mage.abilities.effects.common.cost.SpellCostReductionForEachSourceEffect;
+import mage.abilities.effects.common.cost.SpellsCostReductionAllEffect;
+import mage.constants.*;
+import mage.abilities.keyword.DevoidAbility;
+import mage.abilities.keyword.DeathtouchAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.counters.CounterType;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.permanent.CounterAnyPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetPermanentOrPlayer;
+import mage.util.CardUtil;
+import mage.watchers.common.CastSpellLastTurnWatcher;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class MutatedCultist extends CardImpl {
+
+    public MutatedCultist(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{B}");
+        
+        this.subtype.add(SubType.ELDRAZI);
+        this.subtype.add(SubType.HORROR);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(3);
+
+        // Devoid
+        this.addAbility(new DevoidAbility(this.color));
+
+        // When you cast this spell, remove all counters from up to one target permanent or opponent. The next spell you cast this turn costs {1} less to cast for each counter removed this way.
+        Ability ability = new CastSourceTriggeredAbility(new MutatedCultistEffect());
+        ability.addTarget(new TargetPermanentOrPlayer());
+        this.addAbility(ability);
+        // Deathtouch
+        this.addAbility(DeathtouchAbility.getInstance());
+
+    }
+
+    private MutatedCultist(final MutatedCultist card) {
+        super(card);
+    }
+
+    @Override
+    public MutatedCultist copy() {
+        return new MutatedCultist(this);
+    }
+}
+
+class MutatedCultistEffect extends OneShotEffect {
+
+    private static final FilterPermanent filter = new FilterPermanent("permanent with a counter");
```

Comment:
unused so remove (it is allowed to target something with no counters)

Reply:
removed